docs/projects/automate-project-git-source.ipynb

```diff
@@ -0,0 +1,2069 @@
+{
```

Comment:
<p>You can use the <code>get_function</code> method to <s>both get the function object, and</s> change the function spec - you mention in the first row that you can use is to get the object.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/3640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
updated

Reply:
<p>&nbsp;<a href="https://app.reviewnb.com/mlrun/mlrun/pull/3640/#running-the-function-locally" target="_blank">run jobs locally</a>  and remotely - add  remotely to the line</p><p> </p>